using-mobile-extensions/adobe-analytics-mobile-services/README.md

```diff
@@ -601,22 +471,6 @@ For local \(remote\) notifications, the following metrics are tracked:
 The following example shows you how to include open tracking:
 
 {% tabs %}
-{% tab title="iOS" %}
```

Comment:
Yes, this was intentional. I assumed there already would be one since we created additional tasks for 3.x.  I plan to add new tasks for anything not covered still.